x/group/testutil/app_config.go

```diff
@@ -0,0 +1,12 @@
+package testutil
+
+import (
+	_ "embed"
+
```

Comment:
As aaron mentioned in another, we should add the blank imports of the `app.yaml` defined module here.
See https://github.com/cosmos/cosmos-sdk/pull/12240#discussion_r896140280